main.go

```diff
@@ -37,7 +39,7 @@ var (
 	wrap         = flag.Int("wrap", 100, "column to wrap at")
 	tab          = flag.Int("tab", 8, "tab width for column calculations")
 	overwrite    = flag.Bool("w", false, "overwrite modified files")
-	fast         = flag.Bool("fast", false, "skip running goimports")
+	fast         = flag.Bool("fast", false, "skip running goimports and simplify")
```

Comment:
Can you also update the new usage info in the README?

Reply:
done